lib/queue/consumer.ts

```diff
@@ -131,11 +138,21 @@ export class Consumer implements QueueConsumer {
 				pgPool: this.pool,
 				concurrency: defaultEnvironment.queue.concurrency,
 				pollInterval: 1000,
-				logger: new Logger((_scope) => {
-					return _.noop;
-				}),
+				logger: new Logger(graphileLoggerToJellyfishLogger(logContext)),
 				taskList: {
-					actionRequest: async (result) => {
+					actionRequest: async (result, helpers) => {
```

Comment:
Where does this `helpers` come from?

Reply:
This is how `graphile` passes a logger object. See https://github.com/graphile/worker#create-tasks and https://github.com/graphile/worker#logger.

This part isn't nice, but that's what `graphile` provides for logging.